feature-libs/asm/root/facade/asm-facade.service.ts

```diff
@@ -0,0 +1,18 @@
+import { Injectable } from '@angular/core';
+import { facadeFactory } from '@spartacus/core';
+import { Observable } from 'rxjs';
+import { ASM_FEATURE } from '../feature-name';
+import { BindCartOptions } from '../model/bind-cart.model';
+
+@Injectable({
+  providedIn: 'root',
+  useFactory: () =>
+    facadeFactory({
+      facade: AsmFacadeService,
+      feature: ASM_FEATURE,
+      methods: ['bindCart'],
+    }),
+})
+export abstract class AsmFacadeService {
```

Comment:
To have a consistent naming, the `AsmFacadeService` should be called `AsmFacade` instead